ui-v2/app/templates/dc/services/-upstreams.hbs

```diff
@@ -11,7 +11,13 @@
     {{/block-slot}}
     {{#block-slot 'row'}}
       <td>
-        <span data-test-destination-name>{{item.DestinationName}}</span>
+        <a data-test-destination-name>
```

Comment:
Why is this an anchor?

Reply:
From memory I think I noticed we were using a `a` elsewhere, and I'm pretty sure we are going to link these very soon.